part-spec/unit.json

```diff
@@ -1,76 +1,101 @@
-            "example": "VDD1",
-            "type": "string"
-          },
-          "relativeValueModifier": {
-            "description": "if a unit quantity is based on another reference, the value that edits that reference",
-            "example": "1.2",
-            "type": "number"
-          },
-          "relativeValueOperator": {
-            "description": "if a unit quantity is based on another reference, the operation that is performed with the modifier",
-            "example": "multiply",
-            "type": "string",
-            "enum": [
-              "multiply",
-              "add",
-              "subtract"
-            ]
-          },
-          "valueDefined": {
-            "description": "a boolean representing whether a value has been defined",
-            "type": "boolean"
-          }
-        }
+  "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/unit.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Specifications of unit",
+  "unit": {
+    "type": "object",
+    "properties": {
+      "siUnit": {
+        "description": "name of SI unit of measure",
+        "enum": [
+          "microvolt",
+          "millivolt",
+          "volt",
+          "kilovolt",
+          "microamp",
+          "milliamp",
+          "amp",
+          "milliohm",
+          "ohm",
+          "kilohm",
+          "megaohm",
+          "coulomb",
+          "millihenry",
+          "henry",
+          "picofarad",
+          "nanofarad",
+          "microfarad",
+          "millifarad",
+          "farad",
+          "nanosecond",
+          "microsecond",
+          "millisecond",
+          "second",
+          "milliwatt",
+          "watt",
+          "kilowatt",
+          "joule",
+          "celsius",
+          "micrometer",
+          "millimeter",
+          "meter",
+          "milligram",
+          "gram",
+          "kilogram",
+          "hertz",
+          "kilohertz",
+          "megahertz",
+          "percentage",
+          "bit",
+          "byte"
+        ],
+        "type": "string"
+      },
+      "unitName": {
+        "description": "name of unit if not defined in the siUnit list",
+        "type": "string"
+      },
+      "typValue": {
+        "description": "typical unit quantity corresponding to unit text - example 40mV would have a value of 40",
+        "type": "number"
+      },
+      "minValue": {
+        "description": "minimum unit quantity corresponding to unit text - example 40mV would have a value of 40",
+        "type": "number"
+      },
+      "maxValue": {
+        "description": "maximum unit quantity corresponding to unit text - example 40mV would have a value of 40",
+        "type": "number"
+      },
+      "unitFactor": {
+        "description": "multiplier on the value to achieve the SI unit - example for 40mV the unitFactor would be 0.001",
```

Comment:
We need to clarify that the unitFactor/multiplier is  in referenced to one of the SI units included in the list . In this case the multiplier for 40mV should be 1 since we have mV in the list. We could use an example like 40uV where the unit factor can be 0.001 and the siUnit is mV.

Reply:
would you like to discuss in the working group? Otherwise I can try to modify the description

Reply:
Yes, please go ahead and modify the description.